apps/bemo-worker/wrangler.toml

```diff
@@ -1,5 +1,5 @@
 main = "src/worker.ts"
-compatibility_date = "2024-06-25"
```

Comment:
why down?

Reply:
i was getting warnings that this was newer than the version of wrangler we had installed